build/bake/file-definition.md

```diff
@@ -48,17 +48,17 @@ Complete list of valid target fields available for [HCL](#hcl-definition) and
 
 | Name                | Type   | Description                                                                                                                     |
 |---------------------|--------|---------------------------------------------------------------------------------------------------------------------------------|
-| `inherits`          | List   | [Inherit build options](#merging-and-inheritance) from other targets                                                            |
```

Comment:
`inherits` one was placed on top on purpose if I recall correctly as it's a special field not related with build ones.

Reply:
OK, I think I would still expect to see these args to be in a-z descending order. Do you think we should keep this option at the top?

Reply:
That's fine like this :+1: